credentialz/README.md

```diff
@@ -18,12 +17,11 @@ The following files are expected to be created during the bootstrap process:
   * always required
 * target's private key
   * always required
-* `${system_role_home}/.ssh/authorized_users` file for every system account.  This file contains a list of principals to validate against for access to the system account.
-  * always required
+* `${system_role_home}/.ssh/authorized_principals` file for every system account.  This file contains a list of principals to validate against for access to the system account.
```

Comment:
Got rid of this entry, but I'd say "role" or "system account" are fine. I'd say "local" can be confusing because it could mean "local on the ssh server" or "local on the ssh client"